packages/scenario/test/bind-types.test.ts

```diff
@@ -616,6 +644,62 @@ describe('Data (<bind type>) type support', () => {
 				}
 			);
 		});
+
+		describe('type="date"', () => {
+			let answer: InputNodeAnswer<'date'>;
+
+			beforeEach(() => {
+				answer = getTypedInputNodeAnswer('/root/date-value', 'date');
+			});
+
+			it('has a PlainDate | null static type', () => {
+				expectTypeOf(answer.value).toEqualTypeOf<Temporal.PlainDate | null>();
+			});
+
+			it('has a date populated value', () => {
+				expect(answer.value).to.deep.equal(Temporal.PlainDate.from('2025-12-20'));
+				expect(answer.stringValue).toEqual('2025-12-20');
+			});
+
+			it('has an null as blank value', () => {
+				scenario.answer(inputRelevancePath, 'no');
+				answer = getTypedInputNodeAnswer('/root/date-value', 'date');
+				expect(answer.value).toBeNull();
+				expect(answer.stringValue).toBe('');
+			});
+
+			it.each([
+				'13:30:55',
+				'2025-23-23',
+				'ZYX',
+				'2025-03-07T14:30:00+invalid',
+				'2025-03-07T14:30:00-08:00',
```

Comment:
From PR description: "Datetimes with a timezone offset will result in a null / empty string. We need user research to determine if it should honor the timezone (in local time, it could be one date back or forward) or truncate to the date part (yyyy-mm-dd) as Enketo does."